wrappingGsdk/README.md

```diff
@@ -33,6 +33,12 @@ As mentioned, if you're just evaluating the platform or don't have a game server
 
 You can use the zipped file that was created from `build.ps1` script during your creation of a new MPS Build.
 
+> If PowerShell throws up an error message – <b>File cannot be loaded because running scripts is disabled on this system</b>, then you need to enable the `build.ps1` script to run on your Windows 10 computer.  You can enable this by running the folloiwng PowerShell command in advance of running `build.ps1`:
+
+```
+Set-ExecutionPolicy Unrestricted
```

Comment:
does this need to be run under an admin powershell?

Reply:
Yes, if your script isn't executable, then you must at least make the script executable from an admin Powershell by calling the recommended call above

Reply:
gotcha, do we need to specify that in the instructions/readme then?

Reply:
I'll add it.  Thanks